src/index.html

```diff
@@ -31,24 +31,25 @@
         </header> <!--END OF HEADER-->
 
         <section class="search" ng-controller="Find as find">
+          <div class="color-bar">
             <h2>Find what you're <span id="wish">wishing</span> for</h2>
             <form name="find" ng-submit="search($form)">
-              <input name="query" type="search" placeholder="cast a line" ng-model="query" ng-model-options="{ updateOn: 'default blur'}"></input>
-              <!-- <button type="submit">Search</button> -->
+              <a href="#/results">
+                <input name="query" type="search" placeholder="cast a line" ng-model="query" ng-model-options="{ updateOn: 'default blur'}"></input>
+                <button type="submit">Search</button>
+              </a>
```

Comment:
Sticking this `<input>` tag inside the `<a>` doesn't make a lot of sense to me here... What are you trying to accomplish?